Spot/Spot.swift

```diff
@@ -169,20 +170,24 @@ public protocol SpotDelegate: class {
         return identifier
     }
     
-    static func reportEmailMessageBody() -> String {
+//    static func reportEmailMessageBody() -> String {
+//        return "<Please tell us what happened here>\n\n\(reportInformation())"
+//    }
+    
+    static func reportInformation() -> String {
         let versionNumber = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String
         let buildNumber = Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as! String
         let bundleName = appName()
         
-        var bodyText = "Bundle name: \(bundleName)\nVersion: \(versionNumber)\nBuild: \(buildNumber)\n"
+        var reportInfo = "Bundle name: \(bundleName)\nVersion: \(versionNumber)\nBuild: \(buildNumber)\n"
         if let modelName = Spot.modelName() {
-            bodyText += "Device: \(modelName)"
+            reportInfo += "Device: \(modelName)\n"
         }
         if let additionalInformation = Spot.delegate?.additionalEmailContent() {
-            bodyText += "Additional information: \(additionalInformation)"
+            reportInfo += "Additional information: \(additionalInformation)"
```

Comment:
Worth putting the `\n` on the end in readiness for more being added at any time?